tildetown/templates/frontpage.html

```diff
@@ -0,0 +1,438 @@
+  <li>run <pre>random_pokemon.sh</pre> to get a random ascii art pokemon (thanks, <a href="http://tilde.town/~sanqui">~sanqui</a>)</li>
+  <li>a <a href="http://tilde.town/~wiki">git-based wiki</a> that all townies can edit (thanks for the help, <a href="http://tilde.town/~datagrok">~datagrok</a>)</li>
+  <li>an IRC bot (thanks, <a href="http://tilde.town/~selfsame">~selfsame</a>) that plays numberwang</li>
+  <li>even moar learnings up on the page of <a href="http://tilde.town/~shanx">~shanx</a></li>
+  <li>general good feels</li>
+</ul>
+
+<h4>80 users!</h4>
+<p><sub><em>Wed Oct 29 18:19:45 UTC 2014</em></sub></em>
+
+<p>tilde.town just added its 80th user. Welcome! Some folks have been having
+  trouble with ssh public keys. Remember, public keys are really confusing if you
+  aren't too familiar with them, and there is no judgement if you can't
+  immediately get them to work. Feel free to tweet at <a
+                                                         href="https://twitter.com/tildetown">@tildetown</a> or send an email to <a
+                                                                                                                                    href="mailto:nks@lambdaphil.es">~vilmibm</a> if you'd like some help.</p>
+
+</p>
+
+<h4>60 users</h4>
+<p><sub><em>Sat Oct 25 19:37:47 UTC 2014</em></sub></em>
+<p>tilde.town is up to 60 users. This is lovely! It is also only a tenth of our
+  planned total user capacity, so if you are interested in a tilde* account we'd
+  love to have you. You can even submit a user request with a fancy form now!</p>
+
+<p>
+  <strong><a href="http://goo.gl/forms/8IvQFTDjlo">submit a user account request!</a></strong>
+</p>
+</p>
+
+<h4>Change of IP, warnings when you ssh</h4>
+<p><sub><em>Sun Oct 19 03:28:59 UTC 2014</em></sub></p>
+<p>
+  In the interest of making sure we don't suddenly disappear from the Internet
+  as a result of a dynamic IP change on AWS I switched to using an Elastic
+  IP (amusingly, "elastic" IPs are static).
+</p>
+
+<p>
+  The upside is that we now have a static IP (also it's free) (woo).
+</p>
+
+<p>
+  The downside is that when you try to SSH into tilde.town, now, you'll get an
+  error about "something nasty" and there will be many @ characters. You'll want
+  to edit your .ssh/known_hosts file and remove the entry for tilde.town (your
+  error message should have a cute copy-pasta-able one-liner for doing this for
+  you).
+</p>
+
+<p>
+  Go ahead and do that, and then log in With Confidence. Hopefully this won't happen again.
+</p>
+
+<p>
+  I'll close by mentioning the imminent arrival of many new tildeverse members. We (the tildeverse operators) are just about ready to accept the signups currently languishing in a waitlist. This means lots of new users, which is lovely, but may mean some growing pains while I adapt our server to the new user load in the most cost-effective way.
+</p>
+          </td>
+        </tr>
+      </table>
+      
+      <!--
```

Comment:
ha, oops, this commented out part can be deleted if you don't mind. i forgot it was in there.